build-system/npm-publish/build-npm-binaries.js

```diff
@@ -3,12 +3,17 @@
  * Builds npm binaries for specified Bento components.
  */
 
-const [extension] = process.argv.slice(2);
+const [extensionOrCore] = process.argv.slice(2);
 const {timedExecOrDie} = require('../pr-check/utils');
 const {updatePackages} = require('../common/update-packages');
 
+const shouldBuildCore = !extensionOrCore || extensionOrCore === 'core';
+const ampExtensionsOption = shouldBuildCore
+  ? '--noextensions'
+  : `--extensions=${extensionOrCore}`;
+
```

Comment:
Does this work? I don't see anywhere that passes `core` to this script. 
```
const ampExtensionsOption = extension ? `--extensions=${extensions}` :  '--noextensions'
```

Reply:
sure, done